src/app/components/play-round/play-round.component.scss

```diff
@@ -60,14 +60,13 @@
   }
 }
 
-.marker-name {
+.marker-name, .marker-tip {
```

Comment:
Since you put both the `marker-name` and `marker-tip` divs in a wrapper container, shouldn't we do the positioning and some margins for it on the container?